app/renderer/lib/client-frameworks/robot.js

```diff
@@ -97,6 +97,214 @@ ${this.indent(code, 4)}
   codeFor_swipe (varNameIgnore, varIndexIgnore, x1, y1, x2, y2) {
     return `Swipe    ${x1}    ${y1}    ${x2}    ${y2}`;
   }
+
+  codeFor_getCurrentActivity () {
```

Comment:
FXIME?

Reply:
I'll mention it to the person who put in the PR for the robot framework recording code.